packages/playground/src/components/users-list.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import {
+  Icon,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Text,
+  Avatar,
+  AvatarImage,
+  AvatarFallback,
+  Badge
+} from '@harnessio/canary'
+import { getInitials } from '../utils/utils'
+
+interface UsersProps {
+  display_name: string
+  role: string
+  email: string
+  timestamp?: string
+  avatarUrl?: string
+}
+
+interface PageProps {
+  users: UsersProps[]
+}
+
+export const UsersList = ({ users }: PageProps) => {
+  return (
+    <Table variant="asStackedList" className="border-0">
+      <TableHeader>
+        <TableRow>
+          <TableHead className="text-primary">Name</TableHead>
+          <TableHead className="text-primary">Role</TableHead>
+          <TableHead className="text-primary">Email</TableHead>
+          {users[0]?.timestamp && <TableHead className="text-right text-primary">Date added</TableHead>}
```

Comment:
Why do we have this check on `users[0]?.timestamp`?

Reply:
removed